cluster/addons/cluster-monitoring/influxdb/heapster-controller.json

```diff
@@ -0,0 +1,54 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "ReplicationController",
+  "metadata": {
+    "labels": {
+      "name": "heapster",
+      "kubernetes.io/cluster-service": "true"
+    },
+    "name": "monitoring-heapster-controller"
+  },
+  "spec": {
+    "replicas": 1,
+    "selector": {
+      "name": "heapster"
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "name": "heapster",
+          "kubernetes.io/cluster-service": "true"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "image": "gcr.io/google_containers/heapster:v0.11.0",
+            "name": "heapster",
+            "env": [
+              {
+                "name": "FLAGS",
+                "value": "--poll_duration=2m --stats_resolution=1m --sink influxdb:http://$MONITORING_INFLUXDB_SERVICE_HOST:$MONITORING_INFLUXDB_PORT_8086_TCP_PORT"
```

Comment:
The env variables won't be set for heapster pod unless the influxdb service has been created. Since we start all the monitoring services together, there is no guarantee that that influxdb service will be created before heapster.
Support for multi-port DNS is on the way. until then we can hardcode port 80. WDYT?